pyproject.toml

```
@@ -0,0 +1,95 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "packitos"
+authors = [
+    { name = "Red Hat", email = "user-cont-team@redhat.com" },
+]
+description = "A set of tools to integrate upstream open source projects into Fedora operating system."
+readme = "README.md"
+license = { file = "LICENSE" }
```

hatchling has implemented PEP 639, so this way is preferred (ref: https://hatch.pypa.io/latest/config/metadata/#license)

```suggestion
license = "MIT"
license-files = { paths = ["LICENSE"] }
```